www/src/pages/fr/usage/trpc.md

```diff
@@ -277,24 +277,24 @@ const MyComponent = () => {
   const utils = api.useContext();
   const postCreate = api.post.create.useMutation({
     async onMutate(newPost) {
-      // Cancel outgoing fetches (so they don't overwrite our optimistic update)
+      // Annuler les récupérations sortantes (afin qu'elles n'écrasent pas notre mise à jour optimiste)
       await utils.post.list.cancel();
 
-      // Get the data from the queryCache
+      // Récupérer les données du queryCache
       const prevData = utils.post.list.getData();
 
-      // Optimistically update the data with our new post
+      // Mettez à jour les données de manière optimiste avec notre nouveau message
```

Comment:
I think be can let `post`

in that case, post is a kind of thing

Reply:
Yeah